packages/vue/htmlvalidate/elements/components.js

```diff
@@ -1244,7 +1244,10 @@ module.exports = defineMetadata({
 
     "f-wizard-step": {
         flow: true,
-        requiredAncestors: ["f-wizard > f-wizard-step"],
+        requiredAncestors: [
+            "f-wizard > f-wizard-step",
+            "template > f-wizard-step",
```

Comment:
Tänker att detta är good enough annars om vi strikt bara ska tillåta den som template root i SFC så får vi göra som med modalen.

Nu släpper den tekniskt sett igenom när man slottar in innehåll med `<template #slot-name>` också. Men jag vet inte, dont do that? Fungerar det så vill kanske konsumenten legitimt göra det.